mods/d2k/chrome/ingame-observer.yaml

```diff
@@ -888,8 +879,8 @@ Container@OBSERVER_WIDGETS:
 						ScrollItem@COMBAT_PLAYER_TEMPLATE:
 							X: 0
 							Y: 0
-							Width: 735
 							Height: 25
+							Width: 735
```

Comment:
This should probably be 755? The Width of the stats header looks unedited too.

Reply:
Updated.